README.md

```diff
@@ -1,3 +1,47 @@
 # Plasma Contracts
 
-Root chain contracts for Plasma MVP.
+Root chain contracts for Plasma MVP, work in progress.
+
+## Contents
+This version of the contract implements [Plasma MVP](https://ethresear.ch/t/minimal-viable-plasma/426) (Buterin, Poon, Knott). This implementation PoA scheme with one operator and multiple watchers (users). Detailed description of our child chain design is in [Tesuji document](https://github.com/omisego/elixir-omg/blob/develop/docs/tesuji_blockchain_design.md).
```

Comment:
> This implementation PoA scheme with one operator and multiple watchers (users)

This implementation is a PoA scheme with one operator and multiple watchers (users)

Reply:
:+1: